src/stable/build/commands/compile/javascript.ts

```diff
@@ -19,7 +22,7 @@ function getPrelude(main: string): string {
 
             import { getDefaultVisitorData, IDL, idlToString } from 'azle';
 
-            import * as Canister from './${main}';
+            import * as Canister from '${main}';
```

Comment:
We cannot put the absolute path of the developer into the binary, because the binary will be somewhat public